recipe/conda_build_config.yaml

```diff
@@ -1,6 +1,6 @@
-# New versions need clock_gettime
-# which is supported in 10.12 and later
+# New versions need open_memstream
+# which is supported in 10.13 and later
 macos_min_version:             # [osx and x86_64]
-  - 10.12                      # [osx and x86_64]
+  - 10.13                      # [osx and x86_64]
```

Comment:
did you try maybe increasing this even more?

Reply:
Didn't work unfortunately. I'll try disabling the llvm stuff on OSX (where it's not that relevant anyway as OpenGL can be used)